master/reference/advanced/etcd-rbac/calico-etcdv2-paths.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Calico key path prefixes in etcd v2
+---
+
+The Paths listed here are the path prefixes that a particular calico component
+needs access to in etcd to function successfully.
+
+## calico/node
+
+| Path                       | Access | Notes |
+|----------------------------|--------|-------|
```

Comment:
Should we just get rid of the Notes column?  As it stands, the notes are written in different formats, use abbreviates which probably won't be understood (e.g. WEP), or are missing.